solr/core/src/java/org/apache/solr/update/AddUpdateCommand.java

```diff
@@ -250,6 +261,7 @@ private void recUnwrapRelations(List<SolrInputDocument> unwrappedDocs, SolrInput
     recUnwrapRelations(unwrappedDocs, currentDoc, false);
   }
 
+  /** Extract all anonymous child documents from parent. */
   private void recUnwrapp(List<SolrInputDocument> unwrappedDocs, SolrInputDocument currentDoc, boolean isRoot) {
```

Comment:
Oh; I didn't know -- the docs then suggest to me this method ought to have a clearer name ;-)    `e.g. recUnwrapAnonymous`